### PR TITLE
add AudioContext init config params

### PR DESCRIPTION
**Pull request by HuiGeGeGitHub**
add AudioContext init config params.For example,
 `new window.MediaRecorder(stream, {
      sampleRate: 16000
  })`
That can set audio's sampleRate When init The MediaRecorder

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,13 +31,13 @@ class MediaRecorder {
   /**
    * @param {MediaStream} stream The audio stream to record.
    */
-  constructor (stream) {
+  constructor (stream, config) {
```
Comment (ai) [inline index.js:34]:
```suggestion
  constructor (stream, config = null) {
```

**Comment by ai** [comment]
CI is falling. Try to reduce size or increase limit.

**Comment by HuiGeGeGitHub** [comment]
Package size limit has exceeded by 22 B, Can you increase limit in "size-limit" section of package.json

**Comment by ai** [comment]
1. Try to reduce size by `this.config ? this.config : null` → `this.config`
2. Run `npx size-limit`
3. Put the size to `package.json` in `size-limit` section

**Comment by ai** [comment]
Thanks. Released in 0.4.

**Pull request merged**